### PR TITLE
2.24.0 blog post

### PR DESCRIPTION
**Pull request by petersmythe**
For @jodygarnett to review

Note specifically:

## New Security > URL Checks page

The previous 2.23 series added a new Check URL facility under the Security menu, but it was turned off by default, for backwards compatibility reasons. This functionality allows administrators to manage OGC Service use of external resources.

*Please note, however, that for 2.24.0, the functionality is turned ON by default, to improve GeoServer's default security.*

and

### Revised Security Policy and CVE handling

Jody Garnett (GeoCat) is leading an initiative to work in a more controlled way with security researchers who assess GeoServer for vulnerabilities.
The improvement to the policy can be found in the SECURITY.md file included in every release, and allow us to manage our own CVE numbers.

Please see [GSIP 220](https://github.com/geoserver/geoserver/wiki/GSIP-220) for all the details. We welcome your feedback.

### CODE REVIEW

**Review by mprins — APPROVED** [review]

**Comment by jodygarnett** [comment]
I note that some recent issues were merged and backported, but their associated tickets do not show up in the blog post, or in https://github.com/geoserver/geoserver/releases/tag/2.24.0

**Review by jodygarnett — APPROVED** [review]
Updated blog post, to thank testers and sponsors.
And added security vulnerability placeholders.

File: _posts/2023-07-21-geoserver-2-23-2-released.md [hunk 1]
```diff
@@ -27,6 +27,14 @@ GeoServer  is made in conjunction with GeoTools 29.2, and GeoWebCache 1.23.1.
 
 Thanks to Ian Turton for making this release. 
 
+## Security Considerations
+
+This release addresses security vulnerabilities and is considered an essential upgrade for production systems.
+
+This blog post will be updated in due corse with CVE numbers following our coordinated vulnerability disclosure policy.
```
Comment (mprins) [inline _posts/2023-07-21-geoserver-2-23-2-released.md:34]:
```suggestion
This blog post will be updated in due course with CVE numbers following our coordinated vulnerability disclosure policy.
```

File: _posts/2023-08-30-geoserver-2-22-5-released.md [hunk 2]
```diff
@@ -36,6 +36,14 @@ This GeoServer 2.22.5 maintenance release is final scheduled release of GeoServe
 
 All future releases will require a minimum of Java 11.
 
+## Security Considerations
+
+This release addresses security vulnerabilities and is considered an essential upgrade for production systems.
+
+This blog post will be updated in due corse with CVE numbers following our coordinated vulnerability disclosure policy.
```
Comment (mprins) [inline _posts/2023-08-30-geoserver-2-22-5-released.md:43]:
```suggestion
This blog post will be updated in due course with CVE numbers following our coordinated vulnerability disclosure policy.
```

File: _posts/2023-10-15-geoserver-2-24-0-released.md [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+## Upgrade Notes
+
+GeoServer strives to maintain backwards compatibility allowing for a smooth upgrade experience.
+
+We have one minor change to share this release:
+
+* URL Checks: The [url check security setting](https://docs.geoserver.org/maintain/en/user/security/urlchecks.html) is now enabled by default.
+  
+  In GeoServer 2.22.5 and 2.23.2 this setting was available for use, but was turned off by default.
+  If you are not yet in a position to upgrade to 2.24.0 you may wish to enable recommended setting.
```
Comment (mprins) [inline _posts/2023-10-15-geoserver-2-24-0-released.md:46]:
```suggestion
  If you are not yet in a position to upgrade to 2.24.0 you may wish to enable the recommended setting.
```

File: _posts/2023-10-15-geoserver-2-24-0-released.md [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+
+## Security Considerations
+
+This release addresses security vulnerabilities and is considered an essential upgrade for production systems.
+
+This blog post will be updated in due corse with CVE numbers following our coordinated vulnerability disclosure policy.
```
Comment (mprins) [inline _posts/2023-10-15-geoserver-2-24-0-released.md:52]:
```suggestion
This blog post will be updated in due course with CVE numbers following our coordinated vulnerability disclosure policy.
```

**Review by mprins — COMMENTED** [review]
I'm fixing some typos

File: _posts/2023-10-15-geoserver-2-24-0-released.md [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+[windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.24.0/GeoServer-2.24.0-winsetup.exe/download)), along with 
+[docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.24.0/geoserver-2.24.0-htmldoc.zip/download) and
+[extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.24.0/extensions/).
+
+This is a stable release of GeoServer recommended for production use.
+GeoServer 2.24.0 is made in conjunction with GeoTools 30.0, mapfish-print-v2 2.3.0 and GeoWebCache 1.24.0. 
+
+Thanks to Peter Smythe (AfriGIS) and Jody Garnett (GeoCat) for making this release. 
+
+Thanks to everyone who helped test the release canidate: JP Motaung, Georg Weickelt, Peter Smythe, Tobia Di Pisa, and Giovanni Allegra.
```
Comment (aaime) [inline _posts/2023-10-15-geoserver-2-24-0-released.md]:
```suggestion
Thanks to everyone who helped test the release canidate: JP Motaung, Georg Weickelt, Peter Smythe, Tobia Di Pisa, and Giovanni Allegri.
```

**Review by aaime — APPROVED** [review]
Made a few minor improvements, and made sure 2.24.0 shows up as the new stable series on the home page.

**Comment by petersmythe** [comment]
Thank you all for your valuable input

**Comment by petersmythe** [comment]
> I note that some recent issues were merged and backported, but their associated tickets do not show up in the blog post, or in https://github.com/geoserver/geoserver/releases/tag/2.24.0

I wonder whether this is because there were multiple attempts to release this version, and only the issues in the first one are being referenced.  Something to try: in JIRA, unrelease 2.24.0 and then re-release it?

**Comment by aaime** [comment]
Common cause in my experience is PRs getting merged but Jira ticket not being closed or assigned to the release.

**Comment by jodygarnett** [comment]
> > I note that some recent issues were merged and backported, but their associated tickets do not show up in the blog post, or in https://github.com/geoserver/geoserver/releases/tag/2.24.0
> 
> I wonder whether this is because there were multiple attempts to release this version, and only the issues in the first one are being referenced. Something to try: in JIRA, unrelease 2.24.0 and then re-release it?

The issue was just not closed when the PR was merged and backported.

**Pull request merged**